devices/MAX1704x/Registers.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.MAX1704x
+{
+    /// <summary>
+    /// The Registers class contains constant values representing memory addresses of various registers in the MAX17043 IC.
+    /// </summary>
+    internal static class Registers
+    {
+        // R - 12-bit A/D measurement of battery voltage
+        internal const byte Max17043Vcell = 0x02;
```

Comment:
why are you using const and not Enum? Enum would be much cleaner and much smaller foot print in the memory.

Reply:
Just because I was too lazy to refactor 😅 
Now should be better